openapi.yaml

```
@@ -288,6 +280,7 @@ paths:
         - OI
       summary: Create a note for a trouble ticket
       description: Create a note for a trouble ticket
+      operationId: createMalFaconNote
```

pour garder le meme type de nommage que sur le reste on a pas mis de F maj dans Malfacon

```suggestion
      operationId: createMalfaconNote
```